### DISCUSSION TITLE
My abnt2 keyboard does not have a layout in the keyboard configuration app

### DISCUSSION THREAD

**Discussion by dudu1001** [linuxmint/discussions#314] [Hardware reviews]
### Releases

Linux Mint 21.x

### Compatibility

Works partially

### Description of the hardware

I have a keyboard from Brazil, the layout is called abnt2 and it is a standard that many keyboards here use. for some reason there is no layout compatible with this standard in the keyboard configuration app.
after a quick search I found some people with the same problem, a quick solution is to type `setxkbmap -model abnt2 -layout br` in the terminal which seems to fix the problem. I can probably set up some script that runs this command every time I log in, but that doesn't solve the problem for everyone else.

here is a photo of a keyboard similar to mine here you can see an arrow that indicates a key on the keyboard, in all the layouts I have tested this key is not recognized.

![keyboard problem](https://github.com/linuxmint/discussions/assets/106493868/24488ef6-3adc-4318-a19c-562dbede1233)

**Comment by derthal** [comment]
close the same problem with a dell laptop.

**Reply by derthal** [reply]
but to me it worked: there is default Brazilian layout referred to as "Portuguese (Brazil)", in Linux Mint diagrams of layouts.
<img width="778" height="501" alt="Screenshot from 2026-04-12 13-47-49" src="https://github.com/user-attachments/assets/3efdedb0-e9c4-49c6-8e0f-44ec0dc0f133" />
